10.2/js/first.js

```diff
@@ -0,0 +1,26 @@
+//js file
+
+function Telefon (marka, cena, kolor) {
+	this.marka = marka;
+	this.cena = cena;
+	this.kolor = kolor;
+	var isCheap = false;
+	if(cena < 2000) {
+		isCheap = true;
+		this.isCheap = isCheap;
+	}
```

Comment:
Fajny pomysł na rozszerzenie. Dało się to zapisać trochę łatwiej

`this.isCheap = cena < 2000 ? true : false`